### ISSUE TITLE
Syntax error during generation with shares with contain a dot

### ISSUE THREAD

**Issue by gs4711** [derlin/docker-compose-viz-mermaid#16]
Generated file:
````
%%{init: {'theme': 'default'}}%%
flowchart TB
  VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldb{{./container.mysqldb}} -. /container .-x mysqldb
  VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldbrw{{./container.mysqldb/rw}} x-. /container/rw .-x mysqldb
  VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldbmysqldbdata{{./container.mysqldb/mysqldb-data}} x-. /var/lib/mysql .-x mysqldb
  VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldbmysqldbconfig{{./container.mysqldb/mysqldb-config}} -. /docker-entrypoint-initdb.d .-x mysqldb
  VUsersguentherschreinerprojectsdockerpasswordChangercontainerpasswordChanger{{./container.passwordChanger}} -. /container .-x passwordChanger
  VUsersguentherschreinerprojectsdockerpasswordChangercontainerpasswordChangerrw{{./container.passwordChanger/rw}} x-. /container/rw .-x passwordChanger
  VUsersguentherschreinerprojectsdockerpasswordChangercontainerpasswordChangerpasswordChangerdata{{./container.passwordChanger/passwordChanger-data}} -. /var/www/html/passwordChanger .-x passwordChanger
  passwordChanger --> mysqldb
  P0((3306)) -.-> mysqldb
  P1((81)) -. 80 .-> passwordChanger
  P2((444)) -. 443 .-> passwordChanger
  mysqldb -.- default[/default/]
  passwordChanger -.- default

  classDef volumes fill:#fdfae4,stroke:#867a22
  class VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldb,VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldbrw,VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldbmysqldbdata,VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldbmysqldbconfig,VUsersguentherschreinerprojectsdockerpasswordChangercontainerpasswordChanger,VUsersguentherschreinerprojectsdockerpasswordChangercontainerpasswordChangerrw,VUsersguentherschreinerprojectsdockerpasswordChangercontainerpasswordChangerpasswordChangerdata volumes
  classDef ports fill:#f8f8f8,stroke:#ccc
  class P0,P1,P2 ports
  classDef nets fill:#fbfff7,stroke:#8bc34a
  class default nets

```
````

corrected file (enclose the buggy string into quotes):

````
```mermaid
%%{init: {'theme': 'default'}}%%
flowchart TB
  VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldb{{./container.mysqldb}} -. /container .-x mysqldb
  VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldbrw{{./container.mysqldb/rw}} x-. /container/rw .-x mysqldb
  VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldbmysqldbdata{{./container.mysqldb/mysqldb-data}} x-. /var/lib/mysql .-x mysqldb
  VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldbmysqldbconfig{{./container.mysqldb/mysqldb-config}} -. "/docker-entrypoint-initdb.d" .-x 
mysqldb
  VUsersguentherschreinerprojectsdockerpasswordChangercontainerpasswordChanger{{./container.passwordChanger}} -. /container .-x passwordChanger
  VUsersguentherschreinerprojectsdockerpasswordChangercontainerpasswordChangerrw{{./container.passwordChanger/rw}} x-. /container/rw .-x passwordChanger
  VUsersguentherschreinerprojectsdockerpasswordChangercontainerpasswordChangerpasswordChangerdata{{./container.passwordChanger/passwordChanger-data}} -. /var/w
ww/html/passwordChanger .-x passwordChanger
  passwordChanger --> mysqldb
  P0((3306)) -.-> mysqldb
  P1((81)) -. 80 .-> passwordChanger
  P2((444)) -. 443 .-> passwordChanger
  mysqldb -.- default[/default/]
  passwordChanger -.- default

  classDef volumes fill:#fdfae4,stroke:#867a22
  class VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldb,VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldbrw,VUsersguent
herschreinerprojectsdockerpasswordChangercontainermysqldbmysqldbdata,VUsersguentherschreinerprojectsdockerpasswordChangercontainermysqldbmysqldbconfig,VUsersgu
entherschreinerprojectsdockerpasswordChangercontainerpasswordChanger,VUsersguentherschreinerprojectsdockerpasswordChangercontainerpasswordChangerrw,VUsersguent
herschreinerprojectsdockerpasswordChangercontainerpasswordChangerpasswordChangerdata volumes
  classDef ports fill:#f8f8f8,stroke:#ccc
  class P0,P1,P2 ports
  classDef nets fill:#fbfff7,stroke:#8bc34a
  class default nets

```

````

**Comment by gs4711** [comment]
[
[docker-compose.yaml.txt](https://github.com/user-attachments/files/16417388/docker-compose.yaml.txt)
](url)

**Comment by derlin** [comment]
Hello @gs4711 and thank you for pointing it out! This is indeed a change in Mermaid. I will push a fix soon!

**Comment by derlin** [comment]
I pushed a fix, will release soon.

@gs4711 don't hesitate to let me know if something else is amiss, thanks again.

**Comment by derlin** [comment]
Closed by #17
